src/templates/xywh_panel_template.html

```diff
@@ -33,18 +33,27 @@
                         <label class="col-sm-2 control-label">Width</label>
                         <div class="col-sm-4">
                             <input type="number" name="width" min="0" class="form-control input-sm" value="<%= width %>">
-                        </div>
-                    </div>
-                    <div class="form-group" style="margin-bottom: -15px;">
+                        </div>                     
+                    </div>                      
```

Comment:
The only diff here is adding whitespace. Would be nice to clean that up (as you mentioned).